io/include/pcl/io/tim_grabber.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2020-, Open Perception
+ *  Copyright (c) 2020, ysuzuki19
+ *
+ *  All rights reserved
+ */
+
+#pragma once
+
+#include <pcl/pcl_exports.h>
+#include <pcl/console/print.h>
+#include <pcl/common/time.h>
+#include <pcl/io/grabber.h>
+#include <pcl/point_types.h>
+#include <pcl/point_cloud.h>
+#include <boost/asio.hpp>
+#include <boost/thread.hpp>
+#include <vector>
+#include <string>
+#include <thread>
+
+namespace pcl
+{
+
+class PCL_EXPORTS TimGrabber : public Grabber
```

Comment:
Explain the details here. Move the link to the protocol description here as well